src/ApiCalls.js

```diff
@@ -4,6 +4,19 @@ const ApiCalls = {
     .then(response => response.json())
     .then(data => data)
     .catch(error => console.log(error.message))
+  },
+
+  postUserLogin(userLoginInfo) {
+    return fetch('https://rancid-tomatillos.herokuapp.com/api/v2/login', {
+      method: 'POST',
+      headers: {
+        'Content-Type': 'application/json'
+      },
+      body: JSON.stringify(userLoginInfo)
+    })
+    .then(response => response.json())
+    .then(data => data)
+    .catch(err => console.log('failed', err.message))
```

Comment:
We should probably be doing more with this error message than just console.log. Would love to display something to the user if the information doesn't match what's in the database.